core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -442,9 +444,29 @@ private long getCurrentBlockId() {
    */
   private long getBlockId(long pos) {
     int index = (int) (pos / mBlockSize);
```

Comment:
can this use the new method `getBlockIndex`?

Reply:
done